tests/keras/utils/data_utils_test.py

```diff
@@ -187,7 +188,8 @@ def test_generator_enqueuer_processes():
     acc = []
     for i in range(100):
         acc.append(int(next(gen_output)[0, 0, 0, 0]))
-    assert acc != list(range(100)), "Order was keep in GeneratorEnqueuer with processes"
+    assert acc != list(range(100)), (
+        "Order was keep in GeneratorEnqueuer with processes")
```

Comment:
This was a pre-existing issue, but prefer using `'` as string delimiter for consistency with the rest of the file (likewise for the other string error message below).